src/spec/makePythonCode.js

```diff
@@ -80,11 +81,15 @@ function getBody(parameters) {
     : '';
 }
 
-function getHeaders() {
+function getHeaders(parameters = []) {
   return (
     'headers = {\n' +
-    '    "Accept": "application/json",\n' +
-    '    "Content-Type": "application/json",\n' +
+    Object.entries(
+      getHeadersWithExtras(parameters.filter(parameter => parameter.in === 'header'))
+    ).map(([key, value]) => `      "${key}": "${value}"`)
+     .join(',\n') + '\n' +
+    // '    "Accept": "application/json",\n' +
```

Comment:
can this be removed or is it needed in the future?

Reply:
> Looks good, I'm guessing we just copy/paste the `swagger.json` over the `v12` file anytime we have an API update and all should work as expected?

Correct. I'll add something in Confluence to explain the process, but it should be globally unchanged. The goal is to make doc updates as easy as possible so it can be updated more or less at the same time

Reply:
It would be a really nice step if we let this automatically fetch the `swagger.json` file from our API and only provide the overlay, this will make sure it's always in sync

Reply:
Done in latest commits ;)